.github/workflows/ci.yml

```diff
@@ -0,0 +1,59 @@
+name: CI
+
+on:
+  push:
+    branches: [ main, v0 ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  test:
+    # TODO(preview): other platforms
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.9', '3.10', '3.11', '3.12', '3.13', '3.14']
```

Comment:
Is it really pertinent to test on all of those releases? Looks like for Temporal's Python SDK, we're only testing min and max supported.